src/cmd/ksh93/tests/data/sh_match1.xml

```diff
@@ -167,7 +167,7 @@
         </variablelist>
     </refsect1>
     <refsect1 id="environment">
-        <title>ENVIONMENT</title>
+        <title>ENVIRONMENT</title>
```

Comment:
One of the changes made in this file is causing `sh_match` test to fail:

```
<E> sh_match[169]: test_xmlfragment1/0/testfile1.xml
<E> expect: |$'12822 characters to process...\n#input and output OK (12823 characters).', got $'12824 characters to process...\n#input and output OK (12825 characters).'|
<E> actual: ||
<W> sh_match[462]: pattern #13 ere |^x\!y$| and glob |x\!y| produce diff output
<W> sh_match[462]: pattern #20 ere |^x|y$| and glob |@(x|y)| produce diff output
<W> sh_match[462]: pattern #31 ere |^x\a|b$| and glob |@(xa|b)| produce diff output
<W> sh_match[501]: pattern #13 ere |^x\!y$| and glob |x!y| produce diff output
<W> sh_match[501]: pattern #21 ere |^x|y$| and glob |x|y| produce diff output
<W> sh_match[501]: pattern #34 ere |^z.?a$| and glob |~(E)z.?a| produce diff output
<W> sh_match[501]: pattern #38 ere |^x\a|b$| and glob |x\a|b| produce diff output
<W> sh_match[-1]: error_count = 1
```

Reply:
The test failure is because the length of the text being used has increased by two chars.  Thus changing 12822 to 12824 and 12823 to 12835. I'll fix that and improve the error message.